cmd/check.go

```diff
@@ -13,11 +13,11 @@ import (
 )
 
 var checkFileCmd = &cobra.Command{
-	Use:   "check file {path}",
+	Use:   "file {path}",
```

Comment:
I think it can be ambiguous. I would rather call it `check {path}` or something else, but not `file`

Reply:
Indeed it is called ```file``` but it's a subcommand of check. So in the end it becomes ```check file {path}```